### PR TITLE
Update Feedreader.md

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

File: apps/Feedreader.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ authors:
 
 links:
   - type: Web
-    url: "https://jangernert.github.io/feedreader/"
+    url: "http://jangernert.github.io/FeedReader/"
```
Comment (quassy) [inline apps/Feedreader.md]: if possible, https should always stay ;) (will fix the wrong capitalisation)

File: apps/Feedreader.md [hunk 2]
```diff
@@ -28,11 +30,11 @@ screenshots:
 installation:
   - system: elementary
     info:
-    - version: 0.3
+    - version: 1.4.3
```
Comment (quassy) [inline apps/Feedreader.md]: this field refers to the OS version, not the app version (which would change to quickly to keep track of)

Comment (bilelmoussaoui) [inline apps/Feedreader.md]: Ah i didn't knew, thanks!

**Comment by quassy** [comment]
Thanks for the PR! As I could not merge everything I made the relevant changes with a commit myself (attribution in commit message ;) )

**Comment by bilelmoussaoui** [comment]
Thanks @quassy

**Pull request closed without merging**